in_toto/in_toto_keygen.py

```diff
@@ -113,7 +113,8 @@ def main():
       elif args.type == in_toto.util.KEY_TYPE_ED25519:
         in_toto.util.prompt_generate_and_write_ed25519_keypair(args.name)
       else:  # pragma: no cover
-        log.error("(in-toto-keygen) Unsupported keytype: {0}".format(str(args.type)))
+        LOG.error(
+            "(in-toto-keygen) Unsupported keytype: {0}".format(str(args.type)))
```

Comment:
Side note: I notice we're inconsistent with indexing format parameters. Is it necessary unless we're explicitly messing with the default order? Also, I looked at the code style guidelines and it doesn't speak about `format()` at all, and just about `%s` vs `+` for strings.

Reply:
I think this is okay for now. But feel free to open an issue on the code style repo. ;)

Reply:
For reference, we did have a related discussion in https://github.com/in-toto/in-toto/issues/25.